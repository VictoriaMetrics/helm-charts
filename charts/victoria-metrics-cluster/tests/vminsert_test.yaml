suite: vminsert templates test
chart:
  version: 0.1.1
  appVersion: 0.1.0
templates:
  - vminsert-deployment.yaml
tests:
  - it: deployment should match snapshot
    asserts:
      - matchSnapshot: {}
  - it: deployment should match snapshot with fullnameOverride, extraLabels and podLabels
    set:
      vminsert:
        name: insert
        fullnameOverride: vminsert-node
        extraLabels:
          extraLabelName: extraLabelValue
        podLabels:
          podLabelName: podLabelValue
    asserts:
      - matchSnapshot: {}
