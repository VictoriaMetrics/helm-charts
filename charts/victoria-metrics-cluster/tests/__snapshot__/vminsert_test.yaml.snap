deployment should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app: vminsert
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: victoria-metrics-cluster
        app.kubernetes.io/version: 0.1.0-cluster
        helm.sh/chart: victoria-metrics-cluster-0.1.1
      name: RELEASE-NAME-victoria-metrics-cluster-vminsert
      namespace: NAMESPACE
    spec:
      replicas: 2
      selector:
        matchLabels:
          app: vminsert
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: victoria-metrics-cluster
      template:
        metadata:
          labels:
            app: vminsert
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/managed-by: Helm
            app.kubernetes.io/name: victoria-metrics-cluster
        spec:
          automountServiceAccountToken: true
          containers:
            - args:
                - --envflag.enable
                - --envflag.prefix=VM_
                - --httpListenAddr=:8480
                - --loggerFormat=json
                - --storageNode=RELEASE-NAME-victoria-metrics-cluster-vmstorage-0.RELEASE-NAME-victoria-metrics-cluster-vmstorage.NAMESPACE.svc.cluster.local.:8400
                - --storageNode=RELEASE-NAME-victoria-metrics-cluster-vmstorage-1.RELEASE-NAME-victoria-metrics-cluster-vmstorage.NAMESPACE.svc.cluster.local.:8400
              image: victoriametrics/vminsert:0.1.0-cluster
              imagePullPolicy: IfNotPresent
              livenessProbe:
                failureThreshold: 3
                initialDelaySeconds: 5
                periodSeconds: 15
                tcpSocket:
                  port: http
                timeoutSeconds: 5
              name: vminsert
              ports:
                - containerPort: 8480
                  name: http
              readinessProbe:
                failureThreshold: 3
                httpGet:
                  path: /health
                  port: http
                  scheme: HTTP
                initialDelaySeconds: 5
                periodSeconds: 15
                timeoutSeconds: 5
          serviceAccountName: RELEASE-NAME-victoria-metrics-cluster
deployment should match snapshot with fullnameOverride, extraLabels and podLabels:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app: insert
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: victoria-metrics-cluster
        app.kubernetes.io/version: 0.1.0-cluster
        extraLabelName: extraLabelValue
        helm.sh/chart: victoria-metrics-cluster-0.1.1
      name: vminsert-node
      namespace: NAMESPACE
    spec:
      replicas: 2
      selector:
        matchLabels:
          app: insert
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: victoria-metrics-cluster
      template:
        metadata:
          labels:
            app: insert
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/managed-by: Helm
            app.kubernetes.io/name: victoria-metrics-cluster
            podLabelName: podLabelValue
        spec:
          automountServiceAccountToken: true
          containers:
            - args:
                - --envflag.enable
                - --envflag.prefix=VM_
                - --httpListenAddr=:8480
                - --loggerFormat=json
                - --storageNode=RELEASE-NAME-victoria-metrics-cluster-vmstorage-0.RELEASE-NAME-victoria-metrics-cluster-vmstorage.NAMESPACE.svc.cluster.local.:8400
                - --storageNode=RELEASE-NAME-victoria-metrics-cluster-vmstorage-1.RELEASE-NAME-victoria-metrics-cluster-vmstorage.NAMESPACE.svc.cluster.local.:8400
              image: victoriametrics/vminsert:0.1.0-cluster
              imagePullPolicy: IfNotPresent
              livenessProbe:
                failureThreshold: 3
                initialDelaySeconds: 5
                periodSeconds: 15
                tcpSocket:
                  port: http
                timeoutSeconds: 5
              name: vminsert
              ports:
                - containerPort: 8480
                  name: http
              readinessProbe:
                failureThreshold: 3
                httpGet:
                  path: /health
                  port: http
                  scheme: HTTP
                initialDelaySeconds: 5
                periodSeconds: 15
                timeoutSeconds: 5
          serviceAccountName: RELEASE-NAME-victoria-metrics-cluster
